File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/nu/dto/d/NurDocStructTreeDTO.java
package iih.ci.mr.nu.dto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 护理文书结构化浏览树DTO DTO数据 
 * 
 */
public class NurDocStructTreeDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 主键ID
	 * @return String
	 */
	public String getId() {
		return ((String) getAttrVal("Id"));
	}
	/**
	 * 主键ID
	 * @param Id
	 */
	public void setId(String Id) {
		setAttrVal("Id", Id);
	}
	/**
	 * 显示名称
	 * @return String
	 */
	public String getName() {
		return ((String) getAttrVal("Name"));
	}
	/**
	 * 显示名称
	 * @param Name
	 */
	public void setName(String Name) {
		setAttrVal("Name", Name);
	}
	/**
	 * 父级主键
	 * @return String
	 */
	public String getId_par() {
		return ((String) getAttrVal("Id_par"));
	}
	/**
	 * 父级主键
	 * @param Id_par
	 */
	public void setId_par(String Id_par) {
		setAttrVal("Id_par", Id_par);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.mr/nu/dto/d/NurDocStructTreeDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mr.nu.dto.d
{
    /// <summary>
    /// NurDocStructTreeDTO 的摘要说明。
    /// </summary>
    public class NurDocStructTreeDTO : BaseDTO {

        public NurDocStructTreeDTO() {
 
        }

        /// <summary>
        /// 主键ID
        /// </summary>
		public string Id {
            get { return getAttrVal<string>("Id",null); }
            set { setAttrVal<string>("Id", value); }
        }

        /// <summary>
        /// 显示名称
        /// </summary>
		public string Name {
            get { return getAttrVal<string>("Name",null); }
            set { setAttrVal<string>("Name", value); }
        }

        /// <summary>
        /// 父级主键
        /// </summary>
		public string Id_par {
            get { return getAttrVal<string>("Id_par",null); }
            set { setAttrVal<string>("Id_par", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id", "Name", "Id_par"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.nu.dto.d.NurDocStructTreeDTO";
        }
    }
}
